t/ui/18-tests-details.t

```diff
@@ -151,7 +151,8 @@ subtest 'show job modules execution time' => sub {
 subtest 'displaying image result with candidates' => sub {
     $driver->find_element('[href="#step/bootloader/1"]')->click();
     wait_for_ajax;
-    is_deeply(find_candidate_needles, {'inst-bootmenu' => []}, 'correct tags displayed');
+    my $needles = find_candidate_needles;
+    is(join(' ', keys %$needles), 'inst-bootmenu', 'correct tags displayed');
```

Comment:
I'm still wondering what's wrong with my suggestion?
```suggestion
    is_deeply($needles, {'inst-bootmenu' => []}, 'correct tags displayed') or diag explain $needles;
```
I'm not sure about how important this is for this test, but previously the test did not only check if a certain key exists, but also if the value is an empty array ref.